FRONTEND/src/components/gallery/Gallery.jsx

```diff
@@ -1,65 +1,51 @@
-import React from "react";
-import image1 from "../../assets/Gallery/image1.jpg";
-import image2 from "../../assets/Gallery/image2.jpg"; // Update the path
-import image3 from "../../assets/Gallery/image3.jpg"; // Update the path
-import a from "../../assets/Gallery/a.jpeg"; // Update the path
-import b from "../../assets/Gallery/b.jpeg"; // Update the path
-import d from "../../assets/Gallery/d.jpeg"; // Update the path
-import c from "../../assets/Gallery/c.jpeg"; // Update the path
-import image8 from "../../assets/Gallery/image8.jpeg"; // Update the path
-import image9 from "../../assets/Gallery/image9.jpeg"; // Update the path
+import React, { useState, useEffect } from "react";
+import "./Gallery.css"; // Import CSS file for animations
+
 const Gallery = () => {
+  const [images, setImages] = useState([]);
+
+  useEffect(() => {
```

Comment:
It should remain the same unless the page reloads. But, it is causing trouble when I scroll down

Reply:
Resolved No reloads on Scroll Down

Reply:
okay its done